force-app/main/default/classes/SOQL.cls

```diff
@@ -116,7 +132,7 @@ public with sharing class SOQL {
 	 * @return A list of Case records.
 	 */
 	public static List<Case> getFilterCaseOrigins() {
-		return null; // Replace null with the variable you used to store the result
+		return [SELECT Id, Origin FROM Case WHERE Origin NOT IN ('Web', 'Email') AND Origin != NULL];
```

Comment:
Try to create a list or set and using a bind variable in the query. It's easier to manage and you can change the list or set later on. Also, null is not the same as blank. Blank/empty is the absense of a value while null is a value